actions/mouths-of-watersheds/mow/geo_data_grid_2d/__init__.py

```diff
@@ -0,0 +1,57 @@
+class GeoDataGrid2DError(Exception):
```

Comment:
Worth remarking that this module was copied unmodified from climate-explorer-backend (or wherever) and that it may contain some unused code.